jenkins/ci.suse.de/templates/cloud-ardana-job-dac-3cp-template.yaml

```diff
@@ -1,5 +1,10 @@
 - job-template:
     name: 'cloud-ardana{version}-job-dac-3cp-{arch}'
+    properties:
+      - lockable-resources:
+          label: "{build_pool}"
+          var-name: "clm_env"
```

Comment:
why did we call this clm_env rather than ardana_env ? would be more consistent with the rest of the naming..

Reply:
Done